### ISSUE TITLE
v1.8.1 Cannot find debug.log during testnet sync

### ISSUE THREAD

**Issue by justo4** [cryptoadvance/specter-desktop#1564]
When performing the initial sync of a local pruned testnet node, attempting to view the bitcoind logs via the "See Bitcoind Logs" in the "Debug" section, it seems like there is a pathing issue as the following "server error" shows up
![spectre_error](https://user-images.githubusercontent.com/23394448/152929960-66762e2c-5aac-4168-8db0-f48e95c6cb7a.png)

However, manually drilling into the path shows an additional subdir "testnet3," which is where the debug.log file is kept and is appropriately being updated
![console](https://user-images.githubusercontent.com/23394448/152930796-ec6ab764-7f6d-47c1-b500-9d5e5f57a785.png)

**Comment by k9ert** [comment]
In #1378 it seemed that this is a windows issue. But this looks like gnome, no? That is definitely a bug.

**Comment by justo4** [comment]
Yes, it was gnome in unbuntu. I'll look again to see if it keeps making
further/deeper directories as I've not looked at that debug log for a few
startups since.

**Comment by justo4** [comment]
> That was all the further down it went so may not be the same issue as #1378 ?
